server/Dockerfile

```diff
@@ -29,7 +29,7 @@ USER root
 
 ENV HOME /opt/jboss/
 
-RUN INFINISPAN_SHA=$(curl $DISTRIBUTION_URL.sha1); curl -o /tmp/server.zip $DISTRIBUTION_URL && sha1sum /tmp/server.zip | grep $INFINISPAN_SHA \
```

Comment:
https://github.com/infinispan/infinispan/pull/5969 will add sha1 checksums starting with 9.3.0.Beta1. You can certainly restore the verification